admin_guide/sdn_troubleshooting.adoc

```diff
@@ -0,0 +1,527 @@
+# Here we see the entries for two remote nodes
+table=0, priority=100,arp,arp_tpa=10.1.2.0/24  actions=set_field:192.168.122.18->tun_dst,output:1
+table=0, priority=100, ip, nw_dst=10.1.2.0/24  actions=set_field:192.168.122.18->tun_dst,output:1
+
+table=0, priority=100,arp,arp_tpa=10.1.0.0/24  actions=set_field:192.168.122.202->tun_dst,output:1
+table=0, priority=100, ip, nw_dst=10.1.0.0/24  actions=set_field:192.168.122.202->tun_dst,output:1
+
+# Other traffic destined for a local pod IP that hasn't been handled by a higher priority rule
+# goes out port 9 to the virtual bridge for docker
+table=0, priority=75,  ip, nw_dst=10.1.1.0/24  actions=output:9
+table=0, priority=75, arp,arp_tpa=10.1.1.0/24  actions=output:9
+
+# Then ports 3-8 or 10+ are for local pods, here there are two local pods
+table=0, priority=100, ip, nw_dst=10.1.0.7     actions=output:8
+table=0, priority=100,arp,arp_tpa=10.1.0.7     actions=output:8
+
+table=0, priority=100, ip, nw_dst=10.1.0.10    actions=output:12
+table=0, priority=100,arp,arp_tpa=10.1.0.10    actions=output:12
+....
+
+The SDN networking plugin configures two entries (one for arp and one
+for ip) with `output=1` per peer endpoint (i.e. if there are five
+nodes, then there should be 4 * 2 rules; In the example above we have
+3 nodes total, so there are four entries above).  It also sets up the
+other entries on ports 2 and 9 that are shown above.  If there are
+flows missing, please look in the <<reading-the-logs>> section.
+
+==== Is the `iptables` configuration correct?
+
+Check the output from `iptables-save` to make sure you are not
+filtering traffic.  However, OpenShift sets up iptables rules during
+normal operation, so do not be surprised to see entries there.
+
+
+==== Is your external network correct?
+
+Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
+
+== Debugging Pod Egress
+
+If you are trying to access an external service from a pod, e.g.:
+....
+curl -kv github.com
+....
+
+Make sure that the DNS is resolving correctly:
+....
+dig +search +noall +answer github.com
+....
+
+That should return the IP address for the github server, but check
+that you got back the correct address.  If you get back no address, or
+the address of one of your machines, then you may be matching the
+wildcard entry in yoir local DNS server.
+
+To fix that, you either need to make sure that DNS server that has the
+wildcard entry is not listed as a `nameserver` in your
+`/etc/resolv.conf` _or_ you need to make sure that the wildcard domain
+is not listed in the `search` list.
+
+If the correct IP address was returned, then try the debugging advice
+listed above in <<debugging-local-networking>>.  Your traffic should
+leave the Open vSwitch on port 2 to pass through the `iptables` rules, then
+out the route table normally.
+
+
+== Reading the Logs
+
+Run: `journalctl -u openshift-node.service --boot | less`
```

Comment:
I forget the magic incantation, but this needs to be conditionalized on whether we're building the origin docs or the OSE docs since the service name is different.